datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml

```diff
@@ -149,6 +149,23 @@
       - <*_SUFFIX>
       - <PREFIX_*_SUFFIX>
       - <*_SUBSTRING_*>
-
+- name: ignore_metrics_by_labels
+  description: |
+    A mapping of labels where metrics with matching label key and values are ignored.
+    Use the "*" wildcard to match all label values.
+  value:
+    type: object
+    properties:
+      - name: target_label_key
+        type: string
+      - name: target_label_value_list
+        type: array
+        items:
+          type: string
+      - name: target_label_value_wildcard
+        type: string
+    example:
+      <KEY_1>: [<LABEL_1>, <LABEL_2>]
+      <KEY_2>: '*'
```

Comment:
Feels a bit surprising that `'*'` isn't put into an array type here (resulting in inconsistent data types: either a `'*'` string, or a list of strings). Is it expected? Usually the API is "if the list contains `'*'` then wildcard will be applied", is it what we have in #7822?

Reply:
Updated, with the current implementation wildcard in any form would work.